features/mix-blend-mode.yml

```diff
@@ -0,0 +1,12 @@
+name: mix-blend-mode
+description: The `mix-blend-mode` CSS property sets how an element's content is visually mixed.
```

Comment:
How about:

```suggestion
description: The `mix-blend-mode` CSS property blends an element's content with its background or parent elements.
```

Reply:
I like the "blends" verb, so I accepted the change. I also added example blend modes to convey that there are options.